feat: authentication pages added

rdshinde opened the pull request:
- Signup page added
- Login page added

<img width="1440" alt="Screenshot 2022-02-18 at 5 43 27 PM" src="https://user-images.githubusercontent.com/67017632/154680991-c2dec580-80c2-4521-946a-c604a068048b.png">

<img width="1440" alt="Screenshot 2022-02-18 at 5 47 23 PM" src="https://user-images.githubusercontent.com/67017632/154681517-b17ac676-f782-4fee-afae-e590a0b4a44b.png">

rdshinde committed:
feat: authentication pages added

rdshinde committed:
feat: remember me btn added to login form

Sameony commented on templates/login.html:
```diff
@@ -0,0 +1,160 @@
+    />
+    <!-- Local Stylesheet -->
+    <link rel="stylesheet" href="../style.css" />
+    <title>Swadeshi</title>
```
you can give a more specific title to the page such as login | swadeshi

rdshinde replied:
Ya I will change it.

Sameony reviewed (approved):
Good code
a minor suggestion regarding the titles of the webpages:

kjais1720 commented on templates/login.html:
```diff
@@ -0,0 +1,160 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- FontAwesome CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
```
you can import this cdn in a common CSS file (that you will import in every page, like main.css etc.), then you won't need to import the same link in every page and the structure would look much clean

rdshinde replied:
Ya got it thank you. 🙏

kjais1720 reviewed (approved):
Good use of utility classes and reusable components, just a small suggestion ( as mentioned in the comment ).

rdshinde committed:
refactored: fontawesome cdn moved in stylesheet

- Font awesome CDN moved to style.css
- Refactored card footer and wishlist btn
- refactored titles.
